ferry.go

```diff
@@ -380,7 +380,22 @@ func (f *Ferry) WaitUntilBinlogStreamerCatchesUp() {
 // You will know that the BinlogStreamer finished when .Run() returns.
 func (f *Ferry) FlushBinlogAndStopStreaming() {
```

Comment:
I assume this function is called only once (before the cutover)?

Reply:
yes 👍